files/routable-files/index.js

```diff
@@ -0,0 +1,9 @@
+/*jshint node:true*/
+'use strict';
+
+const EngineAddon = require('ember-engines/lib/engine-addon');
+
+module.exports = EngineAddon.extend({
+  name: '<%= dasherizedModuleName %>'<% if (hasLazyFlag) { %>,
+  lazyLoading: <%= isLazy %><% } %>
```

Comment:
Updated. I was actually thinking about this just as I finished up for the day yesterday.

@trentmwillis / @rwjblue: Should we also update this in the `in-repo-engine` blueprint as well?